color-picker-project/src/components/navbar/Navbar.css

```diff
@@ -0,0 +1,120 @@
+.navbar {
+  background-color: white;
+  padding: 16px;
+  border-bottom: 1px solid #dcdcdc;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+}
+
+.menu-button {
+  background: transparent;
+  border: none;
+  font-size: 1.5rem;
+}
+
+.nav-left,
+.nav-right {
+  display: flex;
+  align-items: center;
+  gap: 24px;
+}
+
+.nav-link {
+  display: none;
+  height: 30px;
+  overflow: hidden;
+  font-weight: 500;
+}
+
+.nav-link span {
+  display: flex;
+  align-items: center;
+  height: 30px;
+}
+
+.nav-link span:first-child {
+  color: #6b7280;
+}
+
+.nav-link span:last-child {
+  color: #6366f1;
+}
+
+.signIn-button,
+.signUp-button {
```

Comment:
I believe we had authentication as a stretch goal, but it probably doesn't hurt to leave the login elements on the page for now.  I just wouldn't prioritize hooking them up to any auth service just yet until we know we'll have time to implement.